examples/application_commands/slash_command_trees.py

```diff
@@ -0,0 +1,44 @@
+import discord
+from discord import app_commands
+
+client = discord.Client()
+
+# initiate our command tree, with the client
+apple_tree = app_commands.CommandTree(client)
+apples_picked = []
+
+@client.event
+async def on_ready():
+
+    # add the commands to our guild
```

Comment:
Perhaps something more descriptive, it's easy to mistake using the `app_commands.commandTree.command()` decorator as actually adding the command to the guild, perhaps something like 
```
# Sync our local tree with Discord
``` 
could be more helpful